docs/src/development/variables.md

```diff
@@ -142,6 +142,7 @@ and whether they're available during builds and at runtime.
 | PLATFORM_TREE_ID          | Yes   | Yes     | The ID of the tree the application was built from, essentially the SHA hash of the tree in Git. Use when you need a unique ID for each build |
 | PLATFORM_PROJECT_ENTROPY  | Yes   | Yes     | A random, 56-character value created when the project is created and then stable throughout the project's life. Can be used for Drupal hash salts, Symfony secrets, and other similar values. |
 | PLATFORM_APP_DIR          | Yes   | Yes     | The absolute path to the application directory. |
+| PLATFORM_SOURCE_DIR       | No    | No      | Equivalent to `PLATFRM_APP_DIR` in the context of a running [source operation](../configuration/app/source-operations.md). The directory contains a writable copy of your repository that you can commit to during the operation located at `/mnt/source`. |
```

Comment:
@chadwcarlson , does this mean to access the repo, the directory is `$PLATFORM_SOURCE_DIR/mnt/source`?

Reply:
It means that the returned value of $PLATFORM_SOURCE_DIR is /mnt/source

Reply:
so cat `$PLATFORM_SOURCE_DIR/.platform.app.yaml`